sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2086,6 +2087,12 @@ def __init__(
         GCP expansion service. Used for STORAGE_WRITE_API method.
       max_insert_payload_size: The maximum byte size for a BigQuery legacy
         streaming insert payload.
+      cdc_writes_with_primary_key: Configure the usage of CDC writes on BigQuery
+        and sets the primary key using the column names from the argument.
+        The destination table (or tables if using dynamic destionations) will
+        be created in case of using CREATE_IF_NEEDED mode, and StorageWrite API
+        at least once mode will be configured. Used for STORAGE_WRITE_API
+        method.
```

Comment:
Consider splitting to two parameters `use_cdc writes` and `primary_key` (for reasons mentioned in a previous comment), and call out that primary key is only needed when setting `CREATE_IF_NEEDED`.

Reply:
done.